### PR TITLE
bump ruby version to 2.2.0

### PR DESCRIPTION
**Pull request by pkurek**

### CODE REVIEW

**Comment by fancyremarker** [comment]
Thanks @pkurek! I'm going to merge this to the ruby-2.2 branch, so that it will be available as `quay.io/aptible/ruby:ruby-2.2`.

File: test/ruby.bats [hunk 1]
```diff
@@ -19,5 +19,5 @@
 
 @test "It should be protected against CVE-2014-2525" {
   run dpkg -s libyaml-dev
-  [[ "$output" =~ "0.1.4-2ubuntu0.12.04.3" ]]
+  [[ "$output" =~ "0.1.4-2ubuntu0.12.04.4" ]]
```
Comment (fancyremarker) [inline test/ruby.bats]: Thanks especially for adding this.

**Comment by fancyremarker** [comment]
Merged separately, building as `quay.io/aptible/ruby:ruby-2.2`. Closing this PR.

**Comment by pkurek** [comment]
:tophat: :+1: thank you good Sir.

File: test/ruby.bats (hunk 1 above)
Comment (chrisberkhout) [inline test/ruby.bats]: @fancyremarker, FYI, that bit came up when rebuilding ruby-2.1 as well.

**Pull request closed without merging**